src/qibolab/platforms/abstract.py

```diff
@@ -82,6 +89,13 @@ def channels(self):
                 yield channel
 
 
+@dataclass
+class QubitPair:
```

Comment:
Here I would add a 1 line docstring even if it is already clear what the class is doing.

Reply:
Thanks for the review, done.